blog.html

```diff
@@ -63,8 +63,9 @@ <h3 class="media-heading">
                     </div>
                     <div class="media-text">
                         {{ post.excerpt }}
+
+                        <a href="{{ post.url }}" class="btn-primary btn-default">Continue...</a>
                     </div>
-                    <a href="{{ post.url }}" class="btn-primary btn-default">Continue...</a>
```

Comment:
This breaks the positions of the continue button on the blog cards.
![image](https://user-images.githubusercontent.com/16579931/99318465-a194a200-281c-11eb-9f4a-5182b5436c83.png)

What it is on prod:
![image](https://user-images.githubusercontent.com/16579931/99318505-b5400880-281c-11eb-922b-bdf07be656e0.png)